vector/src/main/java/im/vector/activity/VectorHomeActivity.java

```diff
@@ -207,6 +213,9 @@
     @BindView(R.id.bottom_navigation)
     BottomNavigationView mBottomNavigationView;
```

Comment:
Can you remove the `mBottomNavigationView`
 here?
(see my comment in vector/src/main/res/layout/activity_home.xml)